examples/digital-identity/README.md

```diff
@@ -0,0 +1,9 @@
+# Share session Example
+
+1. From the [Yoti Hub](https://hub.yoti.com) set the application domain of your app to `localhost:9443`
+1. Set the scenario callback URL to `/profile`
+1. Rename the [.env.example](.env.example) file to `.env` and fill in the required configuration values
+1. Install the dependencies with `npm install`
+1. Start the server `node index.js`
```

Comment:
I think we should update this read me.

Reply:
This indeed must be updated... we can leave it to the end, or do it now.
what do you prefer @dhivya-pattabhiraman ?

Reply:
I thought of updating it at the very end when we have a full-fledged example for share v2 normal session and identity profile check session.